db/cards-table.sql

```diff
@@ -0,0 +1,86 @@
+CREATE TABLE CARD_TYPES (
+  TYPE_ID   NUMBER NOT NULL,
+  TYPE      VARCHAR2(255 BYTE),
+  PRIMARY KEY (TYPE_ID)
+);
+
+COMMENT ON TABLE CARD_TYPES IS 'Contains possible values for card type';
+COMMENT ON COLUMN CARD_TYPES.TYPE_ID IS 'ID of type';
+COMMENT ON COLUMN CARD_TYPES.TYPE IS 'Type name';
+
+INSERT INTO CARD_TYPES (TYPE_ID, TYPE) VALUES (1, 'attack');
+INSERT INTO CARD_TYPES (TYPE_ID, TYPE) VALUES (2, 'skill');
+INSERT INTO CARD_TYPES (TYPE_ID, TYPE) VALUES (3, 'power');
+INSERT INTO CARD_TYPES (TYPE_ID, TYPE) VALUES (4, 'status');
+INSERT INTO CARD_TYPES (TYPE_ID, TYPE) VALUES (5, 'curse');
+
+CREATE TABLE CARD_COLORS (
+  COLOR_ID   NUMBER NOT NULL,
+  COLOR      VARCHAR2(255 BYTE),
+  PRIMARY KEY (COLOR_ID)
+);
+
+COMMENT ON TABLE CARD_COLORS IS 'Contains possible values for card color';
+COMMENT ON COLUMN CARD_COLORS.COLOR_ID IS 'ID of color';
+COMMENT ON COLUMN CARD_COLORS.COLOR IS 'Color name';
+
+INSERT INTO CARD_COLORS (COLOR_ID, COLOR) VALUES (1, 'red');
+INSERT INTO CARD_COLORS (COLOR_ID, COLOR) VALUES (2, 'green');
+INSERT INTO CARD_COLORS (COLOR_ID, COLOR) VALUES (3, 'blue');
+INSERT INTO CARD_COLORS (COLOR_ID, COLOR) VALUES (4, 'colorless');
+
+CREATE TABLE CARD_RARITIES (
+  RARITY_ID   NUMBER NOT NULL,
+  RARITY      VARCHAR2(255 BYTE),
+  PRIMARY KEY (RARITY_ID)
+);
+
+COMMENT ON TABLE CARD_RARITIES IS 'Contains possible values for card rarity';
+COMMENT ON COLUMN CARD_RARITIES.RARITY_ID IS 'ID of rarity';
+COMMENT ON COLUMN CARD_RARITIES.RARITY IS 'Rarity name';
+
+INSERT INTO CARD_RARITIES (RARITY_ID, RARITY) VALUES (1, 'basic');
+INSERT INTO CARD_RARITIES (RARITY_ID, RARITY) VALUES (2, 'common');
+INSERT INTO CARD_RARITIES (RARITY_ID, RARITY) VALUES (3, 'uncommon');
+INSERT INTO CARD_RARITIES (RARITY_ID, RARITY) VALUES (4, 'rare');
+
+CREATE TABLE CARDS (
+  ID            NUMBER NOT NULL,
+  TYPE_ID       NUMBER,
+  NAME          VARCHAR2(255 BYTE),
+  COLOR_ID      NUMBER,
+  RARITY_ID     NUMBER,
+  ENERGY        NUMBER,
+  DESCRIPTION   VARCHAR2(255 BYTE),
+  PRIMARY KEY (ID),
+  FOREIGN KEY (TYPE_ID)   REFERENCES CARD_TYPES(TYPE_ID),
+  FOREIGN KEY (COLOR_ID)  REFERENCES CARD_COLORS(COLOR_ID),
+  FOREIGN KEY (RARITY_ID) REFERENCES CARD_RARITIES(RARITY_ID)
+);
+
+COMMENT ON TABLE CARDS IS 'Contains info on Slay the Spire Cards';
+COMMENT ON COLUMN CARDS.ID IS 'Primary key of CARDS';
+COMMENT ON COLUMN CARDS.TYPE_ID IS 'Reference ID of card type in CARD_TYPES table';
+COMMENT ON COLUMN CARDS.NAME IS 'Card name';
+COMMENT ON COLUMN CARDS.COLOR_ID IS 'Reference ID of card color in CARD_COLORS table';
+COMMENT ON COLUMN CARDS.RARITY_ID IS 'Reference ID of card rarity in CARD_RARITIES table';
+COMMENT ON COLUMN CARDS.ENERGY IS 'Energy cost of card as integer';
+COMMENT ON COLUMN CARDS.DESCRIPTION IS 'Card description';
+
+CREATE SEQUENCE card_instance_seq
+    MINVALUE 1
+    START WITH 1
+    INCREMENT BY 1
+    CACHE 10;
+
+--Sample card insert
```

Comment:
This is fine to have for now, but eventually you'll want any sample data to be inserted from the API. You could create a shell script that runs some `curl` commands to insert some test data. Your API will be responsible for running the SQL code. No need to change anything in this script for now.